packages/rancher-gatekeeper/generated-changes/patch/templates/gatekeeper-controller-manager-deployment.yaml.patch

```diff
@@ -11,12 +11,29 @@
          livenessProbe:
            httpGet:
              path: /healthz
-@@ -109,7 +109,7 @@
+@@ -109,16 +109,20 @@
            readOnly: true
        hostNetwork: {{ .Values.controllerManager.hostNetwork }}
        imagePullSecrets:
 -        {{- toYaml .Values.image.pullSecrets | nindent 8 }}
+-      nodeSelector:
+-        {{- toYaml .Values.controllerManager.nodeSelector | nindent 8 }}
 +        {{- toYaml .Values.images.pullSecrets | nindent 8 }}
-       nodeSelector:
-         {{- toYaml .Values.controllerManager.nodeSelector | nindent 8 }}
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
```

Comment:
if you supplied the nodeSelector in the original solution, it would not contain the selectors for Windows.